magic_pod_demo_app.xcodeproj/project.pbxproj

```diff
@@ -91,12 +100,12 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0730;
-				LastUpgradeCheck = 0730;
+				LastUpgradeCheck = 1030;
 				ORGANIZATIONNAME = TRIDENT;
 				TargetAttributes = {
 					08A5C0421D3CAE0F00D8DD31 = {
 						CreatedOnToolsVersion = 7.3;
-						DevelopmentTeam = F2R6DW797Q;
+						DevelopmentTeam = 9U4PT397Q4;
```

Comment:
Is it better to revert this change?

Reply:
One solution: https://stackoverflow.com/questions/39669661/how-to-prevent-xcode-8-from-saving-development-team-in-pbxproj